requirements.txt

```diff
@@ -1,5 +1,5 @@
 #mantid-framework
 matplotlib
 numpy
-PyQt4
+#PyQt4
```

Comment:
Is this commented because installing the mantid-framework will result in installing the appropriate pyqt version?

Reply:
> Is this commented because installing the mantid-framework will result in installing the appropriate pyqt version?

No, `mantid-framework` shouldn't have any qt dependencies. `mantid-framework` is commented out as well.

I commented `PyQt4` out because it was one of the reasons that the CI was failing:

```
ERROR: Could not find a version that satisfies the requirement PyQt4 (from -r requirements.txt (line 4)) (from versions: none)
ERROR: No matching distribution found for PyQt4 (from -r requirements.txt (line 4))
```

I don't think this package actually exist, `PyQt5` does but not `PyQt4`, once we move to PyQt5 we can update this.

pyqt was already getting installed in the CI as part of `conda` [here](https://github.com/neutrons/RefRed/blob/next/.github/scripts/install.sh#L4).

I think the dependencies need to be looked at as there seems to be some that aren't needed, like `pytest-runner` and `pytest-xdist`. And things like `numpy` are getting installed with conda and then again with pip. 

This PR was just to get the CI working and running on PR's and it took restraint to not rewrite it to just do everything with conda instead of a mix of conda and pip.

Reply:
I was "thinking ahead" when I added `pytest-xdist`, to the bright day when we'll have so many tests that we'll need to run them in multiple cores 😄 
`pytest-runner` was to be used if tests were to be run with `setup.py`, which I understood was to be done. We may change to this in the future?

Reply:
`pytest-runner` has been deprecated for years and we shouldn't be using it https://github.com/pytest-dev/pytest-runner/blob/main/README.rst#deprecation-notice